app/manager-frontend/src/pages/ActivitySource/index.jsx

```diff
@@ -87,42 +110,41 @@ export const ActivitySource = ({ activitySource, action, id }) => {
   };
 
   return (
-    <Container my={8} ref={scrollTarget}>
-      <VStack w="100%" align="stretch" spacing={4} p={4} pb={10}>
+    <Container my={8} ref={scrollTarget} style={{ maxWidth: "850px" }}>
```

Comment:
if we want containers to be consistently this size by default, modify the Container defaults in the theme instead of littering every container with an identical attribute (and risking forgetting it)